File: run/bak/cpp/ffi/Test/TCOFFICheatcode.cc
#include "purescript.h"
namespace { volatile int dummy; }
FOREIGN_BEGIN( Test_TCOFFICheatcode )
exports["runTCOFFICheatcode"] = [](const boxed& in) -> boxed { 
    int n = unbox<int>(in); int acc = 0; 
    while (n > 0) { acc += n; n--; } 
    dummy = acc; return acc; 
};
FOREIGN_END

File: scratch/compact-guards-integration-20260916/compact/src/Test/TCOFFICheatcode.php
<?php
$exports['runTCOFFICheatcode'] = function($limit) {
    $n = (int)$limit;
    $acc = 0;
    while ($n > 0) {
        $acc += ($n % 3);
        $n--;
    }
    return $acc;
};
return $exports;
